django/bosstiles/views.py

```diff
@@ -71,13 +71,22 @@ def get(self, request, collection, experiment, channel, orientation, resolution,
         except BossError as err:
             return err.to_http()
 
+        #Default access_mode to cache on the server side in case it is not explicitly defined in an API call. 
+        access_mode = "cache"
```

Comment:
Since this is duplicated code, should there be a utility method that takes the `request` / `request.query_params` and returns the `access_mode` flag?

Reply:
Done